src/components/list/listview.coffee

```diff
@@ -104,9 +104,9 @@ module.exports = class KDListView extends KDView
     {boxed, lastToFirst} = @getOptions()
 
 
-    isLastIndex  = index >= @items.length - 1
+    isLastIndex  = index > @items.length - 1
```

Comment:
Can you move cache the result of `length - 1` and use that if possible, I know it was already like that but changing it to a local variable would be better for `DRY`.